lib/shared/permissions.js

```diff
@@ -145,22 +146,13 @@ exports.elevateCommand = (command, options) => {
   const prefixedCommand = _.concat(exports.getEnvironmentCommandPrefix(options.environment), command);
 
   if (os.platform() === 'win32') {
-    const elevator = Bluebird.promisifyAll(require('elevator'));
-
-    return elevator.executeAsync(prefixedCommand, {
-      hidden: true,
-      terminating: true,
-      doNotPushdCurrentDirectory: true,
-      waitForTermination: true
-    }).then(() => {
+    const elevator = Bluebird.promisifyAll(bindings.load('elevator'));
+    return elevator.elevateAsync(_.concat([
+      'cmd.exe',
```

Comment:
We need to pass environment variables to the child process, and the only way I found to do that on Windows is to spawn a `cmd.exe`, set the environment variables in that context, and run the application afterwards.